lib/grammers-client/examples/dialogs.rs

```diff
@@ -42,8 +40,8 @@ async fn async_main() -> Result<()> {
         .init()
         .unwrap();
 
-    let api_id = env!("TG_ID").parse().expect("TG_ID invalid");
-    let api_hash = env!("TG_HASH").to_string();
+    let api_id = std::env::var("TG_ID")?.parse().expect("TG_ID invalid");
```

Comment:
Please revert this. The use of `env!` is intentional, because developers should be compiling the values into the hinary.

Reply:
What about add a `.cargo/config.toml`
```toml
[env]
TG_ID = ""
TG_HASH = ""
```
to avoid error message in example files.

Reply:
If that works, and they can still by overriden by the user, sure.